content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,492 @@
+
+So far, so good. Congratulate yourself. The blackbox exporter works and you can manually tell it to query a remote target. You are almost there. Now you need to tell Prometheus to do the queries for us.  
+
+Below you find a minimal prometheus config. It is telling Prometheus to scrape the exporter itself as we did [before](#query-exporter) using `curl 'localhost:9115/metrics'`:
+
+NOTE: If you use Docker for Mac or Docker for Windows, you can’t use `localhost:9115` in the last line, but must use `host.docker.internal:9115`. This has to do with the virtual machines used to implement Docker on those operating systems. You should not use this in production.
+
+`prometheus.yml` for Linux:
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - localhost:9115
+```
+
+`prometheus.yml` for macOS and Windows:
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - host.docker.internal:9115
+```
+
+Now run a Prometheus container and tell it to mount our config file from above. Because of the way networking on the host is addressable from the container you need to use a slightly different command on Linux than on MacOS and Windows.:
+
+<a name=run-prometheus></a>
+
+Run Prometheus on Linux (don’t use `--network="host"` in production):
+
+```bash
+docker \
+  run --network="host"\
+  --mount type=bind,source="$(pwd)"/prometheus.yml,target=/prometheus.yml,readonly \
+  prom/prometheus \
+  --config.file="/prometheus.yml"
+```
+
+Run Prometheus on MacOS and Windows:
+
+```bash
+docker \
+  run -p 9090:9090 \
+  --mount type=bind,source="$(pwd)"/prometheus.yml,target=/prometheus.yml,readonly \
+  prom/prometheus \
+  --config.file="/prometheus.yml"
+```
+
+This command works similarly to [running the blackbox exporter using a config file](#run-exporter).
+
+If everything worked, you should be able to go to [localhost:9090/targets](http://localhost:9090/targets) and see under `blackbox-exporter-metrics` an endpoint with the state `UP` in green. If you get a red `DOWN` make sure that the blackbox exporter you started [above](run-exporter) is still running. If you see nothing or a yellow `UNKNOWN` you are really fast and need to give it a few more seconds before reloading your browser’s tab.
+
+To tell Prometheus to query `"localhost:9115/probe?target=prometheus.io&module=http_2xx"` you add another scrape job where you set the `metrics_path` to `/probe` and the parameters under `params:` in the Prometheus config file `prometheus.yml`:
+
+<a name="prometheus-config"></a>
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - localhost:9115   # for Windows and macOS replace with - host.docker.internal:9115
+
+- job_name: 'blackbox'
```

Comment:
This isn't monitoring blackbox so this is misleading. blackbox_http or similar would be clearer.

Reply:
Good Point.

Reply:
It's still important in terms of label taxonomy, it means something to humans and that's importnat.

Reply:
Yes. Exactly because it does mean something for humans I left it aside.

Reply:
But it's not sufficiently distinguishing, you wouldn't be mixing http and tcp probes under the same job. This name to me could also realistically mean the exporter itself.

Reply:
I propose we discuss this further up at the other job name. Those two issues are connected anyway.

Reply:
Resolving that just to make the view clearer. The issue will be discussed above.